### PR TITLE
update(docs): revises site sidenav and top toolbar styles

### PR DESCRIPTION
**Pull request by robertmesserle**

### CODE REVIEW

**Comment by ThomasBurleson** [comment]
@robertmesserle - can you rebase plz

File: docs/app/js/app.js [hunk 1]
```diff
@@ -361,10 +341,15 @@ function(SERVICES, COMPONENTS, DEMOS, PAGES, $location, $rootScope, $http, $wind
 
   function onLocationChange() {
     var path = $location.path();
+    var defaultObject = {
+      name: "Introduction",
+      url:  "/",
+      type: "link"
+    };
 
     if (path == '/') {
```
Comment (ThomasBurleson) [inline docs/app/js/app.js]: this should be:

```js
var introLink = {
   name: "Introduction",
   url:  "/",
   type: "link"
 };

if ( path == introLink.url ) {

}
```

File: docs/config/template/index.template.html [hunk 2]
```diff
@@ -52,22 +50,22 @@ <h2 class="menu-heading md-subhead" ng-if="section.type === 'heading'" id="headi
...
         </md-button>
         <div layout="row" flex class="fill-height" style="max-width: 864px; margin-left: 8px;">
-          <h2 class="md-toolbar-item md-breadcrumb">
+          <h2 class="md-toolbar-item md-breadcrumb md-headline">
             <span ng-if="menu.currentPage.name !== menu.currentSection.name">
               <span hide-sm hide-md>{{menu.currentSection.name}}</span>
               <span class="docs-menu-separator-icon" hide-sm hide-md style="transform: translate3d(0, 1px, 0)">
                 <span class="md-visually-hidden">-</span>
-                <img src="img/docArrow.png" alt="" aria-hidden="true">
+                <md-icon md-svg-src="img/icons/ic_chevron_right_24px.svg" style="transform:translateY(-2px)"></md-icon>
```
Comment (ThomasBurleson) [inline docs/config/template/index.template.html]: what about `aria-hidden` ?

File: docs/config/template/index.template.html [hunk 3]
```diff
@@ -14,14 +14,12 @@
...
-        <a ng-href="#/" layout="row" flex class="docs-logo">
-          <md-icon md-svg-src="md-menu"></md-icon>
-          <div class="docs-logotype">Material Design</div>
-        </a>
-      </h1>
-    </md-toolbar>
+    <header class="nav-header md-whiteframe-glow-z1">
+      <a ng-href="#/" class="docs-logo">
+        <img src="img/icons/angular-logo.svg"/>
+        <h1 class="docs-logotype md-heading">Angular Material</h1>
```
Comment (ThomasBurleson) [inline docs/config/template/index.template.html]: We should start thinking about **l10n** soon....

File: docs/app/js/app.js [hunk 4]
```diff
@@ -40,29 +40,9 @@ function(SERVICES, COMPONENTS, DEMOS, PAGES, $routeProvider, $mdThemingProvider)
...
-    'A700': '#64dd17',
-    'contrastDefaultColor': 'dark',
-    'contrastLightColors': '800 900',
-    'contrastStrongLightColors': '800 900'
-  });
-
   $mdThemingProvider.theme('default')
       .primaryPalette('indigo')
-      .accentPalette('docs-green');
+      .accentPalette('pink');
```
Comment (ThomasBurleson) [inline docs/app/js/app.js]: `pink` ? I thought we wanted the accent color (for tabs bar, checkbox, etc) to be `green`

File: docs/config/template/index.template.html [hunk 5]
```diff
@@ -75,7 +73,7 @@ <h2 class="md-toolbar-item md-breadcrumb">
           <div class="md-toolbar-item docs-tools" layout="column" layout-gt-md="row">
             <md-button class="md-icon-button"
                        aria-label="View Bower Repo on Github"
-                       ng-if="!menu.currentPage || menu.currentSection.name === 'Getting Started'"
+                       ng-if="!currentComponent.docs.length"
```
Comment (ThomasBurleson) [inline docs/config/template/index.template.html]: Should this be improved to:

```html
ng-if=" !currentComponent.hasDocs() "

File: docs/config/template/index.template.html (hunk 2 above)
Comment (robertmesserle) [inline docs/config/template/index.template.html]: Good catch, missed that.

File: docs/app/js/app.js (hunk 4 above)
Comment (robertmesserle) [inline docs/app/js/app.js]: I wanted to ask about that - the original reason for the green accent color was to tie in the date-picker color (when it was green).  I'll switch it back to the custom green.

File: docs/config/template/index.template.html (hunk 5 above)
Comment (robertmesserle) [inline docs/config/template/index.template.html]: Is that a method that's available?

**Comment by robertmesserle** [comment]
@ThomasBurleson Squashed and updated

**Comment by ThomasBurleson** [comment]
Excellent. I will test on my end; as preparation for Naomi's approval.

**Comment by ThomasBurleson** [comment]
Closed with SHA bd9bbdb

**Pull request closed without merging**